corehq/apps/app_manager/templates/app_manager/partials/settings/multimedia_ajax.html

```diff
@@ -1,18 +1,20 @@
 {% load i18n %}
 {% if multimedia_state.has_media %}
-  <div class="panel panel-appmanager">
-    <div class="panel-heading">
-      <h4 class="panel-title panel-title-nolink">{% trans 'Multimedia Reference Checker' %}</h4>
-    </div>
-    <div class="panel-body">
-      <p>
-        <a id="open_checker" target="_blank" href="{% url "hqmedia_references" domain app.get_id %}">
-          {% blocktrans %}View, upload, and download your application's multimedia{% endblocktrans %}
-          <i class="fa fa-external-link"></i>
-        </a>
-      </p>
+  {% if not is_linked_app %}
+    <div class="panel panel-appmanager">
+      <div class="panel-heading">
+        <h4 class="panel-title panel-title-nolink">{% trans 'Multimedia Reference Checker' %}</h4>
+      </div>
+      <div class="panel-body">
+        <p>
+          <a id="open_checker" target="_blank" href="{% url "hqmedia_references" domain app.get_id %}">
+            {% blocktrans %}View, upload, and download your application's multimedia{% endblocktrans %}
+            <i class="fa fa-external-link"></i>
+          </a>
+        </p>
+      </div>
     </div>
-  </div>
+  {% endif %}
 
   <div class="panel panel-appmanager">
     <div class="panel-heading">
```

Comment:
@orangejenny can you help confirm if there is more that should be skipped for linked app, i see that there is quite a lot in `multimedia_zip_notice` that gets shown including the multimedia checker related stuff that we are avoiding.
this is what i see,

<img width="941" alt="Screenshot 2019-08-06 at 8 35 04 PM" src="https://user-images.githubusercontent.com/3864163/62551483-b77d8e00-b889-11e9-9a0a-abf5f2f419ef.png">

Reply:
I think what you have is good - hide the Multimedia Reference Checker section but show the download section.

You're right that the mm ref checker link shouldn't show in linked apps, but I can fix that separately. We're already showing `multimedia_zip_notice` for linked apps, since it's in the deploy modal.